crates/sui-rosetta/src/operations.rs

```diff
@@ -213,23 +249,33 @@ impl Operations {
         Ok(InternalOperation::WithdrawStake { sender, stake_ids })
     }
 
+    fn from_transaction_block_data(
+        data: SuiTransactionBlockData,
+        input_args: Option<Vec<CallArg>>,
+    ) -> Result<Operations, Error> {
+        let sender = *data.sender();
+        Self::from_transaction(data.transaction().clone(), sender, None, input_args)
+    }
+
     fn from_transaction(
         tx: SuiTransactionBlockKind,
         sender: SuiAddress,
         status: Option<OperationStatus>,
-    ) -> Result<Vec<Operation>, Error> {
+        input_args: Option<Vec<CallArg>>,
```

Comment:
do we need option? Could we pass an empty vec?

Reply:
Was about to replace with an empty vec as per your suggestion but apparently we won't need this property at all. thnx!